shared/data/fingerprint-protection.js

```diff
@@ -291,6 +280,32 @@
         return windowScript
     }
 
+    /**
+     * Temporary storage can be used to determine hard disk usage and size.
+     * This will limit the max storage to 4GB without completely disabling the
+     * feature.
+     */
+    function modifyTemporaryStorage () {
+        const randomFunctionName = 'A' + Math.random().toString(36).substring(2) // random string of alphanumeric chars
+        const script = `
+            if (navigator.webkitTemporaryStorage) {
+                try {
+                    ${randomFunctionName} = navigator.webkitTemporaryStorage.queryUsageAndQuota
+                    navigator.webkitTemporaryStorage.queryUsageAndQuota = function queryUsageAndQuota (callback, err) {
+                        const modifiedCallback = function (usedBytes, grantedBytes) {
+                            const maxBytesGranted = 4 * 1024 * 1024 * 1024
+                            const grantedBytes = Math.min(grantedBytes, maxBytesGranted)
```

Comment:
I think we're redeclaring `grantedBytes` here which is throwing some console errors.

Reply:
Hm. I might have spoken too soon, I'm actually not seeing the permission dialog prompt for temporary storage on wevideo.com once I changed the var name of `grantedBytes`.

Reply:
You were right here - var needed renaming. Updated